fix: revert to original codeready instructions (#381)
aerogear/datasync-starter: walkthroughs/1-exploring-datasync-codeready/walkthrough.adoc

. Go to the "All Services" tab 
. Go to "Online IDE" and select "Open Console"
. Login to the CodeReady Workspaces dashboard
. Choose `NodeJS MongoDB Web Application` from list of the available stacks

> . Put `https://github.com/aerogear/datasync-starter#walkthrough` into repository URL

Reviewer: ```suggestion
. Enter `https://github.com/aerogear/datasync-starter#walkthrough` for the repository URL
```

. In the right corner press down arrow and select `Create and Start Editing`

> . In the new window select `Dev File` tab and copy dev file from 

Reviewer: ```suggestion
. In the new window select the `Dev File` tab and the copy dev file from 
```

https://raw.githubusercontent.com/aerogear/datasync-starter/master/devfile.yaml
. Press `Create`

> . Wait for workspace to start

Reviewer: ```suggestion
. Wait for the workspace to start
```

> . If workspace did not start you will need to go to 

Reviewer: This sentence has no ending

}
----

> === Generating your DataSync Node.JS server and React app

Reviewer: ```suggestion
=== Generating your DataSync Node.JS server and React App
```
